recipes/openmpi/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version="1.10.3" %}
+{% set minor_version=".".join(version.split(".")[:2]) %}
+
+package:
+  name: openmpi
+  version: {{ version }}
+
+source:
+  fn: openmpi-{{ version }}.tar.bz2
+  url: http://www.open-mpi.org/software/ompi/v{{ minor_version }}/downloads/openmpi-{{ version }}.tar.bz2
+  sha1: a3355a5c1c96cf9298ef48061d9db8c2fbd014ea
+
+build:
+  number: 0
+  skip: True   # [win]
+
+test:
+  commands:
+    - mpicc --help
+    - mpif90 --help  # [linux]
+
+about:
+  home: http://www.open-mpi.org/
+  license: 3-clause BSD  (http://www.open-mpi.org/community/license.php)
```

Comment:
I like the freedom of format as long as the proper information is there but I am pretty sure some OCD person/recipe dictator will ask you to change this to `BSD 3-clause`.